pom.xml

```diff
@@ -7,7 +7,7 @@
     <!-- Arifact name and version information -->
     <groupId>net.snowflake</groupId>
     <artifactId>snowflake-ingest-sdk</artifactId>
-    <version>0.10.3</version>
+    <version>0.10.7</version>
```

Comment:
We should be creating a 0.10.3.PATCH based on standard, right?

Reply:
Also, does `v0.10.3-cur-branch` the branch we used for `0.10.3`?

Reply:
It is major.minor.patch so I guess 0.10.7 is correct. 
Also, 0.10.7 is already released, I thought we are not releasing a new version for them.
Should we do a prerelease?